packages/firebase_storage/example/lib/main.dart

```diff
@@ -54,11 +55,24 @@ class _MyHomePageState extends State<MyHomePage> {
     final String bucket = await ref.getBucket();
     final String path = await ref.getPath();
 
+    final File tempFile = new File('${systemTempDir.path}/tmp.txt');
+    if (tempFile.existsSync()) {
+      await tempFile.delete();
+    }
+    await tempFile.create();
+    assert(await tempFile.readAsString() == "");
+    final StorageFileDownloadTask task = ref.writeToFile(tempFile);
+    final int bytes = (await task.future).totalByteCount;
```

Comment:
[nit] Maybe rename to `byteCount` so we don't confuse it with the actual bytes downloaded.